src/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/Pal.Unix/StorePal.cs

```diff
@@ -229,56 +229,30 @@ private static void LoadMachineStores()
 
             foreach (FileInfo file in directoryInfo.EnumerateFiles())
             {
-                byte[] bytes;
-
-                try
-                {
-                    bytes = File.ReadAllBytes(file.FullName);
-                }
-                catch (IOException)
-                {
-                    // Broken symlink, symlink to a network file share that's timing out,
-                    // file was deleted since being enumerated, etc.
-                    //
-                    // Skip anything that we can't read, we'll just be a bit restrictive
-                    // on our trust model, that's all.
-                    continue;
-                }
-                catch (UnauthorizedAccessException)
+                using (SafeBioHandle fileBio = Interop.Crypto.BioNewFile(file.FullName, "rb"))
```

Comment:
The old ReadAllBytes version could throw for reasons listed in the comment.  Those cases don't apply here?  Or the fileBio will just be invalid in those cases and the subsequent Try methods will just return false?